chromadb/test/property/test_embeddings.py

```diff
@@ -413,13 +454,18 @@ def delete_by_ids(self, ids: IDs) -> None:
     )
     def update_embeddings(self, record_set: strategies.RecordSet) -> None:
         super().update_embeddings(record_set)
+
+        if record_set["ids"] is None:
+            raise ValueError("IDs should not be empty")
+        n = len(invariants.wrap(record_set["ids"]))
```

Comment:
Should we use the record set count method?

Reply:
we rely on the number of ids to get the number of items within update and upsert